Userland/Libraries/LibWebView/InspectorClient.cpp

```diff
@@ -536,6 +551,19 @@ void InspectorClient::load_inspector()
     generator.set("COMPUTED_STYLE"sv, generate_property_table("computed-style"sv));
     generator.set("RESOVLED_STYLE"sv, generate_property_table("resolved-style"sv));
     generator.set("CUSTOM_PROPERTIES"sv, generate_property_table("custom-properties"sv));
+
+    AK::StringView display = "block"sv;
+    if (m_is_windowed) {
+        display = "none"sv;
+    }
+
+    AK::String button_element = MUST(String::formatted(R"~~~(
+        <button id="close-inspector-button" style="display: {}" title="Close the Inspector" onclick="inspector.close()"></button>
+)~~~"sv,
```

Comment:
Same here, no `sv` on format string literals. You can also use `auto` on `display` and `button_element`.